Lesson 5 homework.playground/Contents.swift

```diff
@@ -0,0 +1,26 @@
+import UIKit
+
+var dictionary = ["Minsk": ("Capital of Belarus", "Europe"), "Abu-Dhabi": ("Capital of UAE", "Africa"), "Gotham": ("Doesn't exist", "America")]
```

Comment:
Maybe we can call this dictionary `citiesData` or something like that to understand that we store cities and some data for them here?